src/Nethermind/Nethermind.AuRa/Rewards/AuRaRewardCalculator.cs

```diff
@@ -80,9 +124,9 @@ private BlockReward[] CalculateRewardsWithContract(Block block)
             }
 
             var (beneficiaries, kinds) = GetBeneficiaries();
-            var transaction = _contract.Reward(beneficiaries, kinds);
-            _contract.InvokeTransaction(block.Header, _transactionProcessor, transaction, _tracer);
-            var (addresses, rewards) = _contract.DecodeRewards(_tracer.ReturnValue);
+            var transaction = contract.Reward(beneficiaries, kinds);
+            contract.InvokeTransaction(block.Header, _transactionProcessor, transaction, _tracer);
```

Comment:
can we tename InvokeTransaction to Call or similar so it is obvious that we are not changing state?